### PR TITLE
i18n Samoan language support 

### LINKED ISSUE #1890: i18n Samoan language support
- **I'm submitting a...**
   - [ ] bug report
   - [x] feature request
   - [ ] question about the decisions made in the repository
   - [ ] question about how to use this project

- **Summary**
-  Feature: Add i18n Samoan language support 

- **Other information**
- I know this language because I learned it.

- [x] Yes, I'm willing to submit a PR

### PR DESCRIPTION
**Pull request by MrKrishnaAgarwal**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Feature: i18n Samoan language support added

- **Why was this change needed?** (You can also link to an open issue here)
 closes #1890
- For convenience and ease

- **Other information**:
- I know this language

### CODE REVIEW

**Comment by MrKrishnaAgarwal** [comment]
cc @scopsy and @p-fernandez

**Comment by MrKrishnaAgarwal** [comment]
@p-fernandez and @scopsy Please review and merge the PR.
Thankyou!

File: packages/notification-center/src/i18n/languages/sm.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const SI: ITranslationEntry = {
+  translations: {
+    notifications: 'faasilasilaga',
+    markAllAsRead: 'faailoga Mea uma e pei ona Faitau',
+    poweredBy: 'fa amalosia ',
+    settings: 'faatulagaga',
+  },
+  lang: 'sm',
```
Comment (p-fernandez) [inline packages/notification-center/src/i18n/languages/sm.ts:10]: Inverting Google translation (as I don't know samoan) return some funny translations. Are you sure these are accurate translations?

Comment (MrKrishnaAgarwal) [reply packages/notification-center/src/i18n/languages/sm.ts:10]: @P-fernandez
I am 99 percent sure that this is the correct translation becuase one of my friend taught me this language so I know it very well.
and about Google translate, it's not that accurate so we shouldn't always believe on it.
Rest up to you :)

Comment (MrKrishnaAgarwal) [reply packages/notification-center/src/i18n/languages/sm.ts:10]: Also be assured, if I won't be sure or I won't know this language well, I wouldn't make a PR for it, and I never believe on Google Cloud.
I am also learning French but sometimes it shows wrong translations

**Comment by MrKrishnaAgarwal** [comment]
@ainouzgali I would appreciate if you would look in this PR

File: packages/notification-center/src/i18n/languages/sm.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const SI: ITranslationEntry = {
```
Comment (ainouzgali) [inline packages/notification-center/src/i18n/languages/sm.ts:3]:
```suggestion
export const SM: ITranslationEntry = {
```

**Review by ainouzgali — COMMENTED** [review]
Hi @MrKrishnaAgarwal, please take al look at my suggested change. 
Will approve it after that. 
Thank you very much!

**Comment by MrKrishnaAgarwal** [comment]
@ainouzgali Requested changes have been made, kindly take a look now.

**Review by ainouzgali — APPROVED** [review]
💪🏻🚀

**Comment by MrKrishnaAgarwal** [comment]
A check has failed 🙁

**Comment by MrKrishnaAgarwal** [comment]
Thankyou 🙂

**Pull request merged**